dashboard/lib/projects_list.rb

```diff
@@ -42,5 +84,33 @@ def get_project_row_data(student, project, channel_id)
         updatedAt: project_value['updatedAt'],
       }.with_indifferent_access
     end
+
+    def fetch_published_project_types(project_types, limit:, offset: 0)
+      {}.tap do |projects|
+        project_types.map do |type|
+          projects[type] = PEGASUS_DB[:storage_apps].
+            where(state: 'active', project_type: type).
+            exclude(published_at: nil).
+            order(Sequel.desc(:published_at)).
+            limit(limit).
+            offset(offset).
+            map {|project| get_published_project_data project}
```

Comment:
Hey @ashercodeorg , As a follow-on, I want to do a join against the `user_storage_ids` table in pegasus db and the `users` table in **dashboard** db to include the student name (abbreviated) and age in this response. Can you advise on whether this:
1. can be done in Sequel, or
2. should be done by passing raw SQL to Sequel, or
3. if it would be better to just do a single, additional query to dashboard db to avoid writing raw SQL?